qemu/tests/cfg/nvdimm.cfg

```diff
@@ -92,3 +92,17 @@
         - nvdimm_hotplug:
             del mem_devs
             target_mems = mem1
+        - nvdimm_negative:
+            type = nvdimm_negative
+            start_vm = no
+            size_mem = 8G
+            pre_command = "dd if=/dev/zero of=${nv_backend} bs=1M count=8192"
+            variants:
+                - align_max:
+                    align_mem = 4G
+                    error_msg = "the alignment \(0x100000000\) is not supported|the alignment \(0x100000000\) exceeds the expected maximum alignment"
```

Comment:
`the alignment \(0x100000000\) (is not supported|exceeds the expected maximum alignment)` ?